SU-HSDO/suhumsci
# SHS-5113: Editoria11y Customizations

# READY FOR REVIEW

## Summary
- Change Editoria11y tab title: "Outline" → "Heading Outline"
- Change styles of alert button to make text black

## Need Review By (Date)
11/27

## Urgency
medium

## Steps to Test
1. Visit any page as a "Developer" user. You should see the Editoria11y widget in the bottom right corner
2. Click on the widget. You should see a panel with three tabs. The second tab should have the title "Heading Outline" instead of "Outline"
    <img width="339" alt="Screenshot 2023-11-23 at 13 21 41" src="https://github.com/SU-HSDO/suhumsci/assets/3496574/ea35512c-1ccc-4255-8d01-d5712b4f4389">
4. Visit a page with an Editoria11y alert status (red outline widget). If you can't find one, you can create a flexible page and add an empty heading in the Text Area. Confirm that the widget error count text is black instead of red.
    <img width="66" alt="Screenshot 2023-11-23 at 13 25 19" src="https://github.com/SU-HSDO/suhumsci/assets/3496574/1c8dfd83-98c1-4496-a5ee-b022bd6aa56c">

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
